LOC.json

```diff
@@ -19202,43 +19202,56 @@
   }, 
   {
     "receptionRange": 18, 
-    "bearing": 15.408, 
+    "bearing": 15.41, 
     "airportICAO": "KMSY", 
-    "elevation": 3, 
+    "elevation": 0, 
     "name": "ILS-cat-I", 
-    "longitude": -90.244669, 
-    "associatedRunwayNumber": "01", 
+    "longitude": -90.25141380, 
+    "associatedRunwayNumber": "02", 
     "frequency": 111700, 
-    "latitude": 30.0057, 
+    "latitude": 29.98450362, 
     "identifier": "IJFI", 
     "type": 4
   }, 
   {
     "receptionRange": 18, 
-    "bearing": 105.554, 
+    "bearing": 105.56, 
     "airportICAO": "KMSY", 
     "elevation": 3, 
     "name": "ILS-cat-III", 
-    "longitude": -90.248847, 
-    "associatedRunwayNumber": "10", 
+    "longitude": -90.28179514, 
+    "associatedRunwayNumber": "11", 
     "frequency": 109900, 
-    "latitude": 29.988706, 
+    "latitude": 29.99665328, 
     "identifier": "IMSY", 
     "type": 4
   }, 
   {
     "receptionRange": 18, 
-    "bearing": 285.553, 
+    "bearing": 285.56, 
     "airportICAO": "KMSY", 
-    "elevation": 5, 
+    "elevation": 0, 
     "name": "ILS-cat-I", 
-    "longitude": -90.284797, 
-    "associatedRunwayNumber": "28", 
+    "longitude": -90.25097379, 
+    "associatedRunwayNumber": "29", 
     "frequency": 109900, 
-    "latitude": 29.997369, 
+    "latitude": 29.98921969, 
     "identifier": "IHOX", 
     "type": 4
   }, 
+  {
+    "receptionRange": 18, 
+    "bearing": 195.41, 
+    "airportICAO": "KMSY", 
+    "elevation": 0, 
+    "name": "LOC", 
+    "longitude": -90.24551065, 
+    "associatedRunwayNumber": "20", 
+    "frequency": 111700, 
+    "latitude": 30.00305238, 
+    "identifier": "IONW", 
+    "type": 4
+  }, 
```

Comment:
LOC approaches should be added to the bottom of the file with the others. Keep in alphabetical order.